app/src/main/java/com/hyperether/getgoing/ui/fragment/ActivitiesFragment.java

```diff
@@ -266,7 +266,7 @@ private void openActivityDetails() {
 
         mileageRun.setOnClickListener(view -> {
             bundle.putInt(BUNDLE_ACTIVITY_ID, ACTIVITY_RUN_ID);
-            listener.onClick(bundle);
+//            listener.onClick(bundle);
```

Comment:
Why is this commented? Did you check functionality after making this callback nonfunctional?

Reply:
the listener didn't work properly. The problem is resolved.